20230727/s8-1690457241591/README.md

```diff
@@ -0,0 +1,66 @@
+# Scenario 8 - FSPIOP Discovery GET Parties with ALS + Sims + K6 VU scaling - ALS:v14.2.2, scale:1, k6vu:1-5
```

Comment:
@mdebarros ..k6vu 1-5 does it indicate that started with 1vu and increased till 5vus?

Reply:
Correct.